src/index.ts

```diff
@@ -71,9 +71,8 @@ const COMPATIBLE_NODE_MODULES = [
 // replaced with the Deno-specific contents listed below.
 const DENO_IMPORTS_SHIM = `@deno/astro-adapter/__deno_imports.ts`;
 const DENO_IMPORTS =
-  `export { Server } from "https://deno.land/std@${DENO_VERSION}/http/server.ts"
-export { serveFile } from 'https://deno.land/std@${DENO_VERSION}/http/file_server.ts';
-export { fromFileUrl } from "https://deno.land/std@${DENO_VERSION}/path/mod.ts";`;
+  `export { Server, serveFile } from "jsr:@std/http@${STD_VERSION}"
```

Comment:
This increases the amount of downloaded modules. Can you split this into 2 imports?:

```suggestion
  `export { Server } from "jsr:@std/http@${STD_VERSION}/server";
export { serveFile } from "jsr:@std/http@${STD_VERSION}/file-server";
```

Reply:
Done. I forced pushed it to the branch to keep the 2 commits intact (1 commit for updating the version of the std, the other one for using JSR).

Reply:
Thanks!